src/tests/ftest/datamover/posix_preserve_props.py

```diff
@@ -58,7 +58,7 @@ def run_dm_preserve_props(self, tool, cont_type, api):
         pool1.connect(2)
 
         # set the path to read and write container properties
-        self.preserve_props_path = self.tmp + "/cont_props.h5"
+        self.preserve_props_path = join(self.tmp, "/cont_props.h5")
```

Comment:
Any joined entry starting with `/` will become the new root. I.e. This actually just becomes `/cont_props.h5`. You would want to NOT include the slash
```suggestion
        self.preserve_props_path = join(self.tmp, "cont_props.h5")
```

Reply:
```
E.g.
>>> os.path.join("/dir1", "/dir2")
'/dir2'
>>> os.path.join("/dir1", "dir2")
'/dir1/dir2'
```

Reply:
@daltonbohning thanks, good catch. 👍